man/metab.Rd

```diff
@@ -52,7 +52,7 @@ Returns daily time series of gross primary production (GPP), respiration (R), an
 # fake data
 	datetime <- seq(as.POSIXct("2014-06-16 00:00:00", tz="GMT"),
      as.POSIXct("2014-06-17 23:55:00", tz="GMT"), length.out=288*2)
-	do.obs <- 2*sin(2*pi*(1/288)*(1:(288*2))+0.75*pi) + 8 + rnorm(288*2, 0, 0.5)
```

Comment:
this just happened when I built the doc - note that I didn't change metab.R, but that it didn't match metab.Rd until this PR